### PR TITLE
[#86662010] display the relevant name in the email From field

### PR DESCRIPTION
**Pull request by afeld**
For new carts, it's the requester, for new comments, it's the commenter, etc.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class CommentsController < ApplicationController
+  before_filter :authenticate_user!
```
Comment (afeld) [inline app/controllers/comments_controller.rb]: Is there a reason we *wouldn't* want to do this?

**Comment by phirefly** [comment]
Good catch! Thankfully, this is just a show page.

**Comment by phirefly** [comment]
:rocket:

**Comment by afeld** [comment]
Actually, it included the creation of comments, too. I've been meaning to run [Brakeman](http://brakemanscanner.org/) on our app to find more things like this.

**Pull request merged**